<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SSOIwidget</class>
 <widget class="QWidget" name="SSOIwidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>566</width>
    <height>451</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>551</width>
     <height>441</height>
    </rect>
   </property>
   <property name="title">
    <string>SSOI</string>
   </property>
   <widget class="QComboBox" name="Version">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>170</width>
      <height>20</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>1</width>
      <height>20</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>999</width>
      <height>20</height>
     </size>
    </property>
   </widget>
   <widget class="QLabel" name="label_83">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Тип</string>
    </property>
   </widget>
   <widget class="QComboBox" name="SsoiAutoDkUse">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>170</width>
      <height>20</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>1</width>
      <height>20</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>999</width>
      <height>20</height>
     </size>
    </property>
   </widget>
   <widget class="QLabel" name="label_84">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>АвтоДК</string>
    </property>
   </widget>
   <widget class="QComboBox" name="SsoiMOprosVariant">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>150</y>
      <width>170</width>
      <height>20</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>1</width>
      <height>20</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>999</width>
      <height>20</height>
     </size>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_SsoiMOprosVariant">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Тип опроса - БП</string>
    </property>
   </widget>
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>10</y>
      <width>371</width>
      <height>371</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="SSOI">
     <widget class="QGroupBox" name="groupBox_6">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>355</width>
        <height>55</height>
       </rect>
      </property>
      <property name="title">
       <string>Канал</string>
      </property>
      <widget class="SSOIPortWidget" name="SSOI_Port_widget" native="true">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>10</y>
         <width>351</width>
         <height>40</height>
        </rect>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="SSOI_M">
     <widget class="QGroupBox" name="groupBox_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>355</width>
        <height>55</height>
       </rect>
      </property>
      <property name="title">
       <string>Канал 1</string>
      </property>
      <widget class="SSOIPortWidget" name="SSOI_M_Port_widget_1" native="true">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>10</y>
         <width>350</width>
         <height>40</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>70</y>
        <width>355</width>
        <height>55</height>
       </rect>
      </property>
      <property name="title">
       <string>Канал 2</string>
      </property>
      <widget class="SSOIPortWidget" name="SSOI_M_Port_widget_2" native="true">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>10</y>
         <width>350</width>
         <height>40</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_4">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>130</y>
        <width>355</width>
        <height>55</height>
       </rect>
      </property>
      <property name="title">
       <string>Канал 1</string>
      </property>
      <widget class="SSOIPortWidget" name="SSOI_M_Port_widget_3" native="true">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>10</y>
         <width>350</width>
         <height>40</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_5">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>190</y>
        <width>355</width>
        <height>55</height>
       </rect>
      </property>
      <property name="title">
       <string>Канал 1</string>
      </property>
      <widget class="SSOIPortWidget" name="SSOI_M_Port_widget_4" native="true">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>10</y>
         <width>350</width>
         <height>40</height>
        </rect>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>SSOIPortWidget</class>
   <extends>QWidget</extends>
   <header location="global">ssoiportwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
